### PR TITLE
Create a first version of the Readme file

### PR DESCRIPTION
**Pull request by jlaehne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+
+
+# RosettaSciIO
+
+The **Rosetta Scientific Input Output library** aims at providing easy reading and
+writing capabilities in Python for a wide range of
+[scientific data formats](https://hyperspy.org/rosettasciio/supported_formats/index.html). Thus
+providing an entry point to the wide ecosystem of python packages for scientific data
+analysis and computation, as well as an interoperability between different file
+formats. Just as the `Rosetta stone <https://en.wikipedia.org/wiki/Rosetta_Stone>`_
```
Comment (ericpre) [inline README.md:19]:
```suggestion
formats. Just as the [Rosetta stone](https://en.wikipedia.org/wiki/Rosetta_Stone)
```

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+provided a translation between ancient Egyptian hieroglyphs and ancient Greek.
+The RosettaSciIO library originates from the `HyperSpy <https://hyperspy.org>`_
```
Comment (ericpre) [inline README.md:21]:
```suggestion
The RosettaSciIO library originates from the [HyperSpy](https://hyperspy.org)
```

**Review by ericpre — COMMENTED** [review]
Thanks, looks good! The syntax of some links need to be changed from rst to md

**Comment by jlaehne** [comment]
Thanks for correcting it directly.

**Pull request merged**